Dockerfile

```
@@ -1,4 +1,4 @@
-ARG BUILD_IMAGE=weaveworks/eksctl-build:2f512606c94211e915e53063662a4df635c95f92
+ARG BUILD_IMAGE=/weaveworks/eksctl-build:9a1b19c7e0061d026d035409003192eb1abb1232
```

This should be
```suggestion
ARG BUILD_IMAGE=weaveworks/eksctl-build:9a1b19c7e0061d026d035409003192eb1abb1232
```

This is a bug in `update-build-image-tag`. I have opened an issue but we can fix it manually here.